uri_test.go

```diff
@@ -368,3 +368,23 @@ func TestURIDefaultPortAmqps(t *testing.T) {
 		t.Fatal("Default port not correct for amqps, got:", uri.Port)
 	}
 }
+
+func TestURITLSConfig(t *testing.T) {
+	url := "amqps://foo.bar/?certfile=cert.pem&keyfile=key.pem&cacertfile=ca.pem&server_name_indication=example.com"
```

Comment:
I would like to see absolute paths used in this test.

Reply:
Added. However, according to the [specification](https://datatracker.ietf.org/doc/html/rfc3986#section-2) all non US-ASCII characters must be encoded using percentage encoding

Reply:
Thanks!